be/src/exprs/bitmap_functions.cpp

```diff
@@ -675,4 +675,60 @@ StatusOr<ColumnPtr> BitmapFunctions::bitmap_subset_in_range(FunctionContext* con
     return builder.build(ColumnHelper::is_all_const(columns));
 }
 
+StatusOr<ColumnPtr> BitmapFunctions::bitmap_to_binary(FunctionContext* context, const starrocks::Columns& columns) {
+    ColumnViewer<TYPE_OBJECT> viewer(columns[0]);
+
+    size_t size = columns[0]->size();
+    ColumnBuilder<TYPE_VARCHAR> builder(size);
+
+    std::unique_ptr<char[]> buf;
+    size_t cur_buf_size = 0;
+
+    for (int row = 0; row < size; ++row) {
+        BitmapValue* bitmap = viewer.value(row);
+        int serialize_size = bitmap->getSizeInBytes();
+        if (serialize_size > cur_buf_size) {
+            buf.reset(new char[serialize_size]);
+            cur_buf_size = serialize_size;
+        }
+
+        bitmap->write((char*)buf.get());
+        builder.append(Slice(buf.get(), serialize_size));
+    }
+    ColumnPtr col = builder.build(ColumnHelper::is_all_const(columns));
+    if (col->capacity_limit_reached()) {
+        return Status::InternalError("Size of binary column generated by bitmap_to_binary reaches limit (4G)");
```

Comment:
```suggestion
        return Status::InternalError("Size of binary column generated by bitmap_to_binary reaches limit (4G)");
```
This is not a good idea to print 4GB in a hard coding mode. If the limit size is changed someday, the log will be different from the real limit.

Reply:
Good suggestion, fixed. return the detail messages